src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -481,22 +482,23 @@ bool fx_muxer_t::resolve_hostpolicy_dir(host_mode_t mode,
     }
 
     // If it still couldn't be found, somebody upstack messed up. Flag an error for the "expected" location.
-    trace::error(_X("A fatal error was encountered. The library '%s' required to execute the application was not found in '%s'."), LIBHOSTPOLICY_NAME, expected.c_str());
-    if (mode == host_mode_t::muxer && !config.get_portable())
+    trace::error(_X("A fatal error was encountered. The library '%s' required to execute the application was not found in '%s'."),
+        LIBHOSTPOLICY_NAME, expected.c_str());
+    if (mode == host_mode_t::muxer && !portable)
     {
-        if (!pal::file_exists(config.get_path()))
+        if (!pal::file_exists(fx_definitions[0]->get_runtime_config().get_path()))
         {
-            trace::error(_X("Failed to run as a standalone app because there is no '%s' file. If this should be a portable app instead, add that file specifying the appropriate framework."), config.get_path().c_str());
+            trace::error(_X("Failed to run as a standalone app because there is no '%s' file. If this should be a portable app instead, add that file specifying the appropriate framework."), fx_definitions[0]->get_runtime_config().get_path().c_str());
         }
-        else if (config.get_fx_name().empty())
+        else if (fx_definitions[0]->get_runtime_config().get_fx_name().empty())
         {
-            trace::error(_X("Failed to run as a standalone app because there is no framework specified in '%s'. If this should be a portable app instead, specify the appropriate framework in that file."), config.get_path().c_str());
+            trace::error(_X("Failed to run as a standalone app because there is no framework specified in '%s'. If this should be a portable app instead, specify the appropriate framework in that file."), fx_definitions[0]->get_runtime_config().get_path().c_str());
         }
     }
     return false;
 }
 
-fx_ver_t fx_muxer_t::resolve_framework_version(const std::vector<fx_ver_t>& version_list,
+fx_ver_t resolve_framework_version(const std::vector<fx_ver_t>& version_list,
```

Comment:
Why this change?

Reply:
I don't see a corresponding change in the `.h` file.  This doesn't seem necessary, is it?

---
In reply to: [153834891](https://github.com/dotnet/core-setup/pull/3460#discussion_r153834891) [](ancestors = 153834891)

Reply:
Removed that change